app/components/GameScreen/GameScreen.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Board from '../Board/Board';
+import GameButtonsPanel from '../GameButtonsPanel/GameButtonsPanel';
+
+
+export default function GameScreen({ state, handleClicks, }) {
+  console.log('in gameScreen');
+  const {
+    toggleStartButtonPressed,
+    toggleEndButtonPressed,
+    findAndDrawPath,
+    handleClickOnBoard,
+  } = handleClicks;
+  const {
+    colorA,
+    colorB,
+    colorC,
+    board,
+    pathStartingPoint,
+    pathEndingPoint,
+    startButtonPressed,
+    endButtonPressed,
+  } = state;
```

Comment:
Destructure in arguments

Reply:
done.

still feels weird with a signature spreading over 10 lines. 

with arrow functions at least it's sort of clear where the signature ends and the function begins. but, like u explained, arrow functions are for implicit returns. oh well